app/components/preprint-form-authors.js

```diff
@@ -1,7 +1,7 @@
 import Ember from 'ember';
```

Comment:
Please use pod-style components when possible; this will make it easier to add/remove components later as needed.

(similar to the file/folder structure in the components directory of ember-osf)

`ember g component something --pod`

Renaming the existing files to match should also work.